substrate/utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -309,9 +310,9 @@ impl PalletCmd {
 			return Err("No benchmarks found which match your input.".into())
 		}
 
-		if self.list {
+		if self.list || self.list_pallets {
```

Comment:
I think it would be good to have `--list=pallets` instead of `--list-pallets`.  
So there could be an enum argument for the `--list` flag that defaults to `all` but can be set to `pallets` as well.